docs/android/geofencing/002.json

```diff
@@ -1,49 +1,44 @@
 {
     "$schema": "http://json-schema.org/schema#",
-    "id": "https://schemas.datacubed.com/android/geofencing/001.json",
+    "id": "https://schemas.datacubed.com/android/geofencing/002.json",
     "index": "https://schemas.datacubed.com/index.json",
     "type": "array",
     "items":    {
         "type": "object",
         "properties": {
-          "geofenceName": {
+          "id": {
             "type": "string"
           },
-          "locationLatitude": {
-              "type": "number",
-              "minimum": -90,
-              "maximum": 90
-          },
-          "locationLongitude": {
-              "type": "number",
-              "minimum": -180,
-              "maximum": 180
+          "name": {
+            "type": "string"
           },
-          "locationAccuracy": {
-              "type": "number",
-              "minimum": 0
+          "event_type": {
+            "type": "string"
           },
           "timestamp": {
               "$ref": "https://schemas.datacubed.com/definitions.json#/timestamp_def"
           },
-          "geofenceLatitude": {
+          "last_updated": {
+              "$ref": "https://schemas.datacubed.com/definitions.json#/timestamp_def"
+          },
+          "latitude": {
               "type": "number",
               "minimum": -90,
               "maximum": 90
           },
-          "geofenceLongitude": {
+          "longitude": {
               "type": "number",
               "minimum": -180,
               "maximum": 180
           },
-          "eventType": {
-            "type": "string"
-          },
-          "geofenceRadius": {
+          "radius": {
               "type": "number",
               "minimum": 0
+          },
+          "snoozed": {
```

Comment:
Let's make 'snoozed' optional so our solution is expandable for other geofence use cases as well.

Reply:
check line 42 now